mvcc/kvstore_test.go

```diff
@@ -513,6 +513,52 @@ func TestRestoreContinueUnfinishedCompaction(t *testing.T) {
 	t.Errorf("key for rev %+v still exists, want deleted", bytesToRev(revbytes))
 }
 
+func TestRestoreContinueUnfinishedCompaction2(t *testing.T) {
```

Comment:
probably we should just combine this test case with `TestRestoreContinueUnfinishedCompaction`. in a table drive style test. the only difference is how we do the restore (create a new one as current TestRestoreContinueUnfinishedCompaction, and call s.store as this test).

basically

``` go
tests := []string{"recreate", "restore"}

for _, t := range tests {
...
    switch t {
        case "recreate": ...
        case "restore": ...
    } 
...
}
```